### DISCUSSION TITLE
Undesirable Alert message box displayed

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#2895] [Q&A]
See image below.  For a particular imodel I occasionally get an annoying Alert message box displayed from my viewer.  For situations like this it would be much better if these could all be console.log messages, especially since there isn't any action I can take to correct the problem.  

![image](https://user-images.githubusercontent.com/30668739/145648446-9b435e99-7a63-479a-b981-d63f80df7942.png)

**Comment by tcobbs-bentley** [comment]
You should be able to disable that alert with the following code:

```ts
ToolAdmin.exceptionOptions.alertBox = false;
```

I believe it already logs the error to console by default, but the following controls that:

```ts
ToolAdmin.exceptionOptions.log = true;
```

**Comment by pmconne** [comment]
The messagebox indicates a serious problem. If you tell us how to reproduce it we can fix it.
